### PR TITLE
Fix issue #37 showing input == null! on eclipse

### PR DESCRIPTION
**Pull request by Mskat**
Folder resources has been created in src.
Moved folder icon to folder resources

### CODE REVIEW

File: src/simplejavacalculator/BufferedImageCustom.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public Image imageReturn()
       throws IOException {
       Image image;
       
-      InputStream bis = getClass().getResourceAsStream("/icon/icon.png");
+      InputStream bis = getClass().getResourceAsStream("/Resources/icon/icon.png");
```
Comment (pH-7) [inline src/simplejavacalculator/BufferedImageCustom.java:17]: Shouldn't `Resources/` be lowercase (`resources/`)?

Comment (Mskat) [reply src/simplejavacalculator/BufferedImageCustom.java:17]: Yes, it should be.

Comment (pH-7) [reply src/simplejavacalculator/BufferedImageCustom.java:17]: Because in your code, it isn't. The first letter (`R`) is capitalized :)

Comment (pH-7) [reply src/simplejavacalculator/BufferedImageCustom.java:17]: Still not sure how you got this error by the way. Are you running the project from the root directory? Normally, you shouldn't get any problem with the icon located in `/icon/ folder outside of the `src` folder.

**Comment by pH-7** [comment]
By the way, I'm till not sure how you got this error. Are you running the project from the root directory? Normally, you shouldn't get any problem with the icon located in `/icon/ folder outside of the `src` folder.

**Comment by Mskat** [comment]
Yes, sorry I didn't see it before. I run the program from src folder.

**Comment by xdvrx1** [comment]
Hello. Good day.

This is my contribution. When it generates error, it means you are in the wrong directory. When you don't manage the codebase as a project, you should place the resources in the `src` folder or create a project with your IDE and make sure that the working directory is the top level of the folder project.

Happy coding!

Visit my projects also <https://github.com/xdvrx1?tab=repositories>

thanks,

**Comment by pH-7** [comment]
> Hello. Good day.
> 
> This is my contribution. When it generates error, it means you are in the wrong directory. When you don't manage the codebase as a project, you should place the resources in the `src` folder or create a project with your IDE and make sure that the working directory is the top level of the folder project.
> 
> Happy coding!
> 
> Visit my projects also https://github.com/xdvrx1?tab=repositories
> 
> thanks,

Thanks for your comment @xdvrx1 Always appreciated 😊

**Pull request merged**